cfgov/jinja2/v1/_includes/organisms/item-introduction.html

```diff
@@ -54,7 +53,7 @@
     {% endif %}
         {% if has_authors %}
             <span class="byline">
-            {%- for author in page_authors -%}
+            {%- for author in page.authors.order_by('name') -%}
                 {{- 'By ' if loop.index == 1 else ' and ' }}
```

Comment:
Oh, sorry, just saw the conversation on this.  I would suggest still using the `get_authors` or `alphabetize_authors` method somehow (or refactoring that as needed) since we also sort by first name if last names match, and it'll be good to keep this logic in one place.